build.gradle

```diff
@@ -4,21 +4,25 @@ buildscript {
             // Code formatting; defines targets "spotlessApply" and "spotlessCheck".
             // https://github.com/diffplug/spotless/tags ; see tags starting "gradle/"
             // Only works on JDK 11+.
-            classpath 'com.diffplug.spotless:spotless-plugin-gradle:6.22.0'
+            classpath 'com.diffplug.spotless:spotless-plugin-gradle:6.23.0'
         }
     }
 }
 
 plugins {
     id "java"
     id "maven-publish"
+    // https://github.com/tbroyer/gradle-errorprone-plugin
+    // id 'net.ltgt.errorprone' version '3.1.0'
```

Comment:
Should this be uncommented? There are some other mentions of errorprone in the build file, so is it correctly running?